main/mainTemplate.json

```diff
@@ -110,7 +109,7 @@
     },
     "virtualNetworkAddressPrefix": {
```

Comment:
`prefix` means the portion after `/`.  the  correct name would be `virtualNetworkAddress` or `virtualNetworkAddresswithPrefix`

Reply:
This is addressPrefix for virtualNetwork.